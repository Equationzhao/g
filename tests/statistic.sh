output="$(g -term-width 200 --no-config --no-update --icons --statistic tests/test_data)"
echo "$output" | tail -n +2 | diff - <(tail -n +2 tests/statistic.stdout)

