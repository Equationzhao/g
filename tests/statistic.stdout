  underwent [0;34m216 microseconds[0m
  statistic: 28 file(s), 1 dir(s), 1 link(s)
[0;94m ?.dir[0m                                                      [0;37m 1.4.1[0m                                                      [38;5;158m 2.c[0m                                                        
[0;37m '   space file '[0m                                           [38;5;153m 1.txt[0m                                                      [38;5;158m 3.cpp[0m                                                      
[0;37m 0.1.112[0m                                                    [0;37m 10.json[0m                                                    [38;5;158m 4.py[0m                                                       
[0;37m 0.1.13[0m                                                     [0;37m 100KiB[0m                                                     [38;5;158m 5.java[0m                                                     
[0;37m 0.1.2[0m                                                      [0;37m 11.xml[0m                                                     [0;92m 6.sh[0m                                                       
[0;37m 0.2.10[0m                                                     [0;37m 12.yml[0m                                                     [38;5;158m 7.html[0m                                                     
[0;37m 0.2.2[0m                                                      [0;93m 13.md[0m                                                      [38;5;158m 8.css[0m                                                      
[0;37m 0.3.0[0m                                                      [0;95m 14.gif[0m                                                     [38;5;158m 9.js[0m                                                       
[0;37m 0.3.84[0m                                                     [0;37m 2.19.1[0m                                                     [0;37m 9.js~[0m                                                      
[0;37m 1.0.1[0m                                                      [0;37m 2.9.1[0m                                                      [0;35m 1.txt.link[0m[0;97m => [0m[38;5;153m/Users/equationzhao/g/tests/test_data/1.txt[0m  
